server/__db/db-config.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
```

Comment:
Would be worthwhile returning a config based on the current environment so you don't have to do it at the point you import the config.

Something like:
```js
const config = {
  production: require('./production'),
  development: require('./development'),
};

const env = process.env.NODE_ENV;

// could add a check here to make sure a config exists for the env if desired
module.exports = config[env];
```

Reply:
I'd also suggest 
a) moving this into the src folder as it's config for the app not the db
b) making it a generic `config.js` file